src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final int ZERO_ELEMENT = 0;
+    private static final int MAX_ELEMENT = Integer.MAX_VALUE;
+    private static final int MIN_ELEMENT = Integer.MIN_VALUE;
+
+    private static Calculator calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void commonAddition() {
+        assertEquals(22, calculator.calculator(20, 2, '+'));
+        assertEquals(-31, calculator.calculator(-30, -1, '+'));
+        assertEquals(19, calculator.calculator(20, -1, '+'));
+        assertEquals(-5, calculator.calculator(-10, 5, '+'));
+        assertEquals(Integer.MAX_VALUE,calculator.calculator(MAX_ELEMENT, ZERO_ELEMENT, '+'));
+        assertEquals(Integer.MIN_VALUE,calculator.calculator(MIN_ELEMENT, ZERO_ELEMENT, '+'));
+    }
+
+    @Test
+    void commonSubtraction() {
```

Comment:
```suggestion
    void subtraction_Ok() {
```
and so on, maybe you may want to reconsider your tests' names

Reply:
Renamed